docs/spas.md

```diff
@@ -0,0 +1,56 @@
+# Single Page Apps
+
+Single Page Apps (aka SPAs) are pages that we render, that are _not_ documents.
+For example the `404` page. These are rendered with server-side rendered but
+without a "doc".
+
+To build these you run:
+
+```sh
+yarn build --spas-only
+```
+
+By default, running only `yarn build` will build all documents _and_ all SPAs.
+You can also opt to _not_ build them and only build the documents. To do that use:
+
+```sh
+yarn build --no-spas
+```
+
+The SPAs are also built when run `yarn prepare-build` or `yarn start`.
+
+Below we describe each SPA and this is a work in progress.
```

Comment:
```suggestion
Below follows the documentation for each SPA. 

 > NOTE: This is still a work in progress.
```

Reply:
I actually prefer my original one since it carries more information.